scripts/install.sh

```diff
@@ -723,11 +723,11 @@ function setup_config_darwin() {
     write_sumologic_extension "${COMMON_CONFIG_PATH}" "${INDENTATION}"
 
     # fill in api base url
-    if [[ -n "${API_BASE_URL}" && -z "${USER_API_URL}" ]]; then
+    if [[ -n "${API_BASE_URL}" ]]; then
```

Comment:
Do we need this change? If yes, can we linux with it?

Reply:
This is necessary on macOS as `common.yaml` is replaced on upgrades. I haven't been able to find a way around this. I'll update the logic for Linux to match.